sample-queries/sample-queries.json

```diff
@@ -1823,6 +1823,17 @@
         "postBody": "{\r\n  \"applicationName\": \"LinkedResource created from Microsoft Graph Explorer\"\r\n}",
         "tip": "This query requires the Tasks.ReadWrite permission. To find a value for the taskListId parameter, you can run: GET https://graph.microsoft.com/beta/me/todo/lists. to find a value for the taskId, you can run: GET https://graph.microsoft.com/beta/me/todo/lists/{taskListId}",
         "skipTest": false
-    }
+    },
+    {
+        "id": "",
+        "category": "Identity and Access",
+        "method": "GET",
+        "humanName": "Count the users in your tenant",
```

Comment:
Maintaining the general convention, the `humanName` value starts with lowercase.